invenio_rdm_records/resources/serializers/csl/__init__.py

```diff
@@ -36,6 +37,7 @@ def __init__(self, **options):
             format_serializer_cls=JSONSerializer,
             object_schema_cls=CSLJSONSchema,
             list_schema_cls=BaseListSchema,
+            schema_kwargs={"dumpers": [JournalCSLDumper()]},  # Order matters
```

Comment:
When tackling the refactoring this would be made configurable, because the dumpers should only be present if the fields are enabled. At the moment this fields are enabled in ZenodoRDM.